apps/storybook/src/Alert.stories.tsx

```diff
@@ -19,9 +19,9 @@ export default {
   },
 } as Meta<AlertProps>;
```

Comment:
`argTypes` and `Meta` can also be removed

Reply:
Removed